File: chinatelcom/ct_consumer/src/main/java/com/soap/chinatelcom/consumer/utils/PropertiesUtil.java
package com.soap.chinatelcom.consumer.utils;

import com.sun.org.apache.bcel.internal.util.ClassLoader;

import java.io.IOException;
import java.io.InputStream;
import java.util.Properties;

/**
 * Created by yangf on 2018/3/20.
 */
public class PropertiesUtil {

    public static Properties properties = null;

    static {
        InputStream resourceAsStream = ClassLoader.getSystemResourceAsStream("hbase_kafk_consumer.properties");
        try {
            properties = new Properties();
            properties.load(resourceAsStream);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }


    public static String getPropertiesKey(String key){
           return properties.getProperty(key);
    }

    

}

File: traffic/traffic_consumer/src/main/scala/com/soap/traffic_sonsumer/util/PropertiesUtil.scala
package com.soap.traffic_sonsumer.util

import java.util.Properties

/**
  * Created by soap on 2018/4/11.
  */
object PropertiesUtil {

  private val properties = new Properties()

  properties.load(ClassLoader.getSystemResourceAsStream("kafka.properties"))

  def getKey(key:String,defaultValue:String = ""):String ={
    properties.getProperty(key,defaultValue)
  }

  def getPeroperties():Properties = {
    properties
  }

}
